### ISSUE TITLE
new term: reduced insulitis severity

### ISSUE THREAD

**Issue by LaurensWilming** [obophenotype/mammalian-phenotype-ontology#2339]
>1. Preferred term label 
**reduced insulitis severity**

>3. Textual definition 
**level of insulitis is lower compared to untreated or wild type organisms**

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.
**PMID:28396406 page E3477 (last paragraph results)**

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
**MP:0004031**

**Comment by sbello** [comment]
As insulitis is a grandchild of increased inflammatory response we do not want to put 'reduced insulitis severity' under this term. Annotate to 'decreased inflammatory response' (MP:0001876).
